core/server/apps/amp/lib/helpers/amp_content.js

```diff
@@ -81,11 +158,7 @@ function ampContent() {
         // errors in video, because video will be stripped out.
         // @TODO: remove this, when Amperize support video transform
         $('video').children('source').remove();
-
-        // Vimeo iframe e. g. come with prohibited attributes
-        // @TODO: remove this, when Amperize supports HTML sanitizing
-        $('amp-iframe').removeAttr('webkitallowfullscreen');
-        $('amp-iframe').removeAttr('mozallowfullscreen');
+        $('video').children('track').remove();
```

Comment:
Bummer 😔  
`amp-*` in allowed tags didn't work, therefore, `amp-*: ['*']` in not necessary and make also `$('*').removeAttr('style')` redundant. I changed it to only allow our supported or AMPs built in tags and will push these changes.